how-to/integrate-with-ms365/client/src/integrations/ms365/integration-provider.ts

```diff
@@ -229,6 +229,17 @@ export class Microsoft365Provider implements IntegrationModule<Microsoft365Setti
 
 		await this.connectProvider();
 
+		const theme = await this._integrationHelpers.getCurrentTheme();
```

Comment:
This is similar to the sales studio approach. One thing we will need to think about is when the ability to switch from light to dark is introduced we need to enable integrations to be updated so that they can update once more.